build/common.targets

```diff
@@ -24,6 +24,15 @@
     <DefineConstants>$(DefineConstants);IS_SIGNING_SUPPORTED</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TFMsBeforeNet5 Condition=" '$(TargetFramework)' == 'net472' OR '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp2.1'">true</TFMsBeforeNet5>
+    <TFMsBeforeNet5 Condition=" '$(TFMsBeforeNet5)' != 'true'">false</TFMsBeforeNet5>
```

Comment:
you don't need the false condition. MSBuild only has string data types, not booleans.

For the condition, instead of comparing exact short tfm names, can you do:

` '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and [MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '5.0'))`

Reply:
I am under the impression that some of the constants should be defined by the SDK. https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md#what-about-net-10

cc @dsplaisted

Not sure where in the SDK they are usually implemented.

Reply:
Thanks. Fixed.